### ISSUE TITLE
Doesn't work in .js file while using JSX in React render method.

### ISSUE THREAD

**Issue by jake-daniels** [formulahendry/vscode-auto-close-tag#14]
I have a file like this

_PageController.js_
```
import React, {Component} from 'react';

class PageController extends Component {

    render() {
        return (
            <div className="page-controller">
                {this.props.children}             
            </div>
        );
    }
}
```

When I try to type new tag inside of the main `div`, extension doesn't work properly.
I type `<h2` and when I press `>`, instead of desired result 

`<h2>{cursor}</h2>`

I end up with 

`<h2</div>{cursor}`

I tried to overwrite settings and add "javascript" and "javascriptreact" to supported languages, but nothing changed.

**Comment by formulahendry** [comment]
Could not repro on my side. I guess maybe some extension else has impact on this function like this [issue](https://github.com/formulahendry/vscode-auto-close-tag/issues/7) (Vim extension affects the Auto Close Tag). Could you please share what other extensions you have used?

**Comment by jake-daniels** [comment]
Just a few. Here's a list.

![untitled](https://cloud.githubusercontent.com/assets/14806713/20028961/df68e1ce-a33f-11e6-901e-675571667952.png)

**Comment by formulahendry** [comment]
Could you please try to put these content into a plain text (e.g. test.txt) and try the same steps to see the result?

**Comment by jake-daniels** [comment]
@formulahendry I already found the solution :)

I use Windows and SK (Slovak) keyboard. On SK, CZ and probably some more keyboard layouts the character `>` is typed as `AltGr + .`. `AltGr` is often interpreted as `Ctrl + Alt` by many applications. So when I tried to type `>`, VS Code interpreted it as keyboard shortcut `{ "key": "ctrl+alt+.", "command": "auto-close-tag.closeTag" }`. I have overwritten the `keybindings.json` file and set en empty command. Now it works perfectly. 

I noticed this while trying to reproduce the issue in plain text file as you suggested and I was unable to type `>` character. At that moment I realized, there must be some kind of shortcut bound to this. So thanks for the help, I think I can close the issue now.
